eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs

```diff
@@ -493,155 +538,160 @@ conwayPParamsPairs pp =
 emptyConwayPParams :: forall era. Era era => ConwayPParams Identity era
 emptyConwayPParams =
   ConwayPParams
-    { cppMinFeeA = Coin 0
-    , cppMinFeeB = Coin 0
-    , cppMaxBBSize = 0
-    , cppMaxTxSize = 2048
-    , cppMaxBHSize = 0
-    , cppKeyDeposit = Coin 0
-    , cppPoolDeposit = Coin 0
-    , cppEMax = EpochNo 0
-    , cppNOpt = 100
-    , cppA0 = minBound
-    , cppRho = minBound
-    , cppTau = minBound
+    { cppMinFeeA = THKD (Coin 0)
```

Comment:
Lots of redundant parentheses here

Reply:
Thank you for pointing it out. Result of search and replace. I'll fix it

Reply:
Removed